### ISSUE TITLE
JAWS not reading title attribute expansion on <abbr> element 

### ISSUE THREAD

**Issue by devarshipant** [FreedomScientific/VFO-standards-support#453]
### On arrowing down using JAWS 2020.2004.66 / Chrome Edge 85.0.4183, the title attribute on the `abbr` element is not read.

[HTML 5.2](https://www.w3.org/TR/html52/textlevel-semantics.html#the-abbr-element) states: The abbr element represents an abbreviation or acronym, optionally with its expansion. The title attribute may be used to provide an expansion of the abbreviation. The attribute, if specified, must contain an expansion of the abbreviation, and nothing else.

> Example:
> 1.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr#:~:text=The%20HTML%20Abbreviation%20element%20(%20%3Cabbr,full%20description%20and%20nothing%20else. -- check out first demo example.

### Expected result

> On arrowing down, the expanded text (via title attribut) should be read.

JAWS 2020.2004.66 / Chrome Edge 85.0.4183 / Win 10 / 64 bit

**Comment by JAWS-test** [comment]
The output of abbreviations must be activated in Settings Management (INS+6). Then it will work. By default it is no longer activated.

**Comment by devarshipant** [comment]
That works! Seems like an important setting that should be enabled as part of default configuration.

**Comment by JAWS-test** [comment]
This used to be enabled by default in JAWS. Why this is no longer the case, I do not know.

**Comment by stevefaulkner** [comment]
@devarshipant may be helpful [short note: The abbreviation appreciation society](https://developer.paciellogroup.com/blog/2019/03/short-note-the-abbreviation-appreciation-society/)
